### DISCUSSION TITLE
CSS Modules

### DISCUSSION THREAD

**Discussion by a-kennedy-develops** [jhipster/generator-jhipster#26881] [Q&A]
Is there any plan or known method to implement CSS Modules in a project utilizing React? I have been struggling to get it working within the given webpack configuration.

**Comment by TatyOko28** [comment]
Yes, CSS Modules can be used in a React project, but it depends on how the webpack configuration is generated/managed.

For a JHipster React project, the usual approach is to add support for files like:

```text
*.module.css
*.module.scss
